README.md

```diff
@@ -67,15 +67,17 @@ e.g.
 ```js
 const { validate } = require('@openactive/data-model-validator');
 
-const data = {
-  // ...
-};
+async function run() {
```

Comment:
remove these wrapper "run" functions from documentation. I don't think it's useful to a developer.